recipes/iotaa/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.7.2" %}
+{% set build = "0" %}
+{% set python = "python >=3.8" %}
```

Comment:
Please don't template these. They obfuscate the recipe and don't help much.

Reply:
Fixed.